src/types.ts

```diff
@@ -7,10 +7,7 @@ export interface NodeData<T = any> {
   parent?: string;
   ports?: PortData[];
   icon?: IconData;
-  paddingTop?: number;
-  paddingLeft?: number;
-  paddingRight?: number;
-  paddingBottom?: number;
+  nodePadding?: [number];
```

Comment:
I don't think this is the right type - I think it should be like: 

```
[number] | [number, number] | [number, number, number, number]
```

Reply:
Correct, I noticed this flaw aswell and addressed it in Commit 8d996a0c142390fd7c3272e807288d8843a04c54. 

According to your [example](https://github.com/reaviz/reaviz/blob/master/src/common/utils/dimensions.ts#L30), I thought of the following type: `number | [number, number] | [number, number, number, number]`

So, if its just one number, it should be just a number and not an array. Otherwise the `parsePadding` method won't work.

Reply:
Oh duh - sorry lol

Reply:
No worries 👍